docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

```diff
@@ -12,7 +12,7 @@ You can use a volume component to share files among container components and col
 
 .Procedure
 
-* Define a `volume` component in your devfile.
+. Define a `volume` component in your devfile and specify a volume size for it.
 +
 .A minimal `volume` component
```

Comment:
Do we want to include the `minimal `volume` component` example?  If so, we should state that default size is determine by the tools.  

```
- name: mydevfile
  volume: {}
```

Reply:
@kim-tsao so keep both steps to the procedures, but only include a yaml example for the ephemeral volume component?

Reply:
I simply removed it. No need to bombard users with multiple yaml examples unless each example helps the user in its own, unique way. Good suggestion, thanks :)